ts-packages/web/src/assets/css/globals.css

```
@@ -172,15 +238,79 @@
+  --color-setting-card-border: #e5e5e5;
+  --color-modal-card-bg: #fafafa;
+  --color-modal-card-border: #d4d4d4;
+  --color-modal-label-text: #a1a1a1;
+
+  --color-label-color-bg: #fcb300;
+  --color-label-color-border: #fcb300;
+  --color-label-color-text: #ffffff;
+  --color-label-bg: #ffffff;
+  --color-label-border: #404040;
+  --color-label-text: #404040;
+  --color-input-box-bg: #fafafa;
```

Biome (2.1.2)

[error] Duplicate custom properties can lead to unexpected behavior and may override previous declarations unintentionally.

--color-input-box-bg is already defined here.

Remove or rename the duplicate custom property to ensure consistent styling.

(lint/suspicious/noDuplicateCustomProperties)